src/models/enums/MessageCodes.ts

```diff
@@ -32,4 +33,6 @@ export enum MessageCodes {
 	ERROR_SIGNING_JWT = "ERROR_SIGNING_JWT",
 	UNEXPECTED_ERROR_FETCHING_OIDC_TOKEN = "UNEXPECTED_ERROR_FETCHING_OIDC_TOKEN",
 	SESSION_EXPIRED = "SESSION_EXPIRED",
+	FAILED_TO_SEND_EMAIL = "FAILED_TO_SEND_EMAIL",
+	UNRECOGNISED_EMAIL_TYPE_UNABLE_TO_BUILD_GOVNOTIFY_MESSAGE = "UNRECOGNISED_EMAIL_TYPE_UNABLE_TO_BUILD_GOVNOTIFY_MESSAGE",
```

Comment:
I think we just need the cause here, we can probably drop the effect, so just `UNRECOGNISED_EMAIL_TYPE`